### PR TITLE
generator@v1.9.0 - Update Template to Support F-Globalisation and F-Development-Context

### PR DESCRIPTION
**Pull request by tommcclean**
> Superpower the generator

**Changed**
- Updated demo to use new format script which uses `f-development-context` to be externalised meaning OOTB `$logger`, `$cookies`, `I18n` and `Vuex`!
- Wire up `F-Globalisation` within the Skeleton and tests so you can use `<i18n>`, `$t('message')` and `this.i18n`
- Fix an existing issue that results in the generator not executing (somebody removed yeoman by accident)
- Fix an issue with the test script - (somebody removed it by accident)

I have tested this by loading the generator component in...
- Storybook (and playing with Locale)
- Yarn Build
- Yarn Test
- Yarn Demo

### CODE REVIEW

File: packages/generator-component/generators/app/templates/src/components/Skeleton.vue [hunk 1]
```diff
@@ -1,34 +1,32 @@
 <template>
     <div
-        :data-theme="theme"
+        data-theme="jet"
```
Comment (ashleynolan) [inline packages/generator-component/generators/app/templates/src/components/Skeleton.vue]: Can delete this line now – it's not needed anymore.

**Review by ashleynolan — COMMENTED** [review]
Looks good – one tiny comment

File: packages/generator-component/generators/app/templates/src/components/Skeleton.vue (hunk 1 above)
Comment (tommcclean) [reply packages/generator-component/generators/app/templates/src/components/Skeleton.vue]: Cheers, will do

**Review by DamianMullins — APPROVED** [review]

**Comment by tommcclean** [comment]
Paused for now due to Storybook Modifications

**Pull request closed without merging**